http_check/changelog.d/19765.added

```diff
@@ -0,0 +1 @@
+Enabled HA Support
```

Comment:
The target audience for the changelogs is customers/users. Imagine you're a customer/user. Do you understand this entry? ;)